contracts/main/stablecoin-core/BookKeeper.sol

```diff
@@ -194,11 +194,19 @@ contract BookKeeper is IBookKeeper, ICagable, IPausable, BookKeeperMath, Pausabl
     }
 
     function setAccessControlConfig(address _accessControlConfig) external onlyOwner {
+        require(_accessControlConfig.isContract(), "BookKeeper/access-control-config: NOT_CONTRACT_ADDRESS");
+        require(
+            IAccessControlConfig(_accessControlConfig).hasRole(IAccessControlConfig(_accessControlConfig).OWNER_ROLE(), msg.sender),
+            "BookKeeper/msgsender-not-owner"
+        );
```

Comment:
I think setAccessControlConfig may not need this require statement because of onlyOwner.

Reply:
this "require" statement check is "msg.sender" has an owner role with new accessControlConfig, meanwhile "onlyOwner" modifier checks with the current one. this is a sanity check to be sure that we set proper contract

Reply:
oh I see. I understood. Thanks for the explaination.